### PR TITLE
DBZ-1546 Remove whitespaces between table.whitelist regexes

### PR DESCRIPTION
**Pull request by grantcooksey**
The table whitespace parameter should remove whitespace between regular expressions
since tables cannot start with whitespace characters.

https://issues.jboss.org/browse/DBZ-1546

### CODE REVIEW

File: debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlConnectorIT.java [hunk 1]
```diff
@@ -993,6 +993,33 @@ public void shouldReceiveSchemaForNonWhitelistedTablesAndDatabases() throws SQLE
         stopConnector();
     }
 
+    @Test
+    @FixFor("DBZ-683")
```
Comment (jpechane) [inline debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlConnectorIT.java]: Is the fixfor correct?

File: debezium-connector-sqlserver/src/test/java/io/debezium/connector/sqlserver/SpecialCharsInNamesIT.java [hunk 2]
```diff
@@ -48,6 +49,75 @@ public void after() throws SQLException {
         }
     }
 
+    @Test
+    @FixFor("DBZ-1546")
```
Comment (jpechane) [inline debezium-connector-sqlserver/src/test/java/io/debezium/connector/sqlserver/SpecialCharsInNamesIT.java]: I like that you've chosen this test class. But could you please use the test schema/table from other method that has space inside its name?

File: debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlConnectorIT.java (hunk 1 above)
Comment (grantcooksey) [reply debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlConnectorIT.java]: Nope, good spot!

File: debezium-connector-sqlserver/src/test/java/io/debezium/connector/sqlserver/SpecialCharsInNamesIT.java (hunk 2 above)
Comment (grantcooksey) [reply debezium-connector-sqlserver/src/test/java/io/debezium/connector/sqlserver/SpecialCharsInNamesIT.java]: 👍

**Comment by jpechane** [comment]
@grantcooksey Applied, thanks!

**Comment by grantcooksey** [comment]
Appreciate the help!

**Pull request merged**